package.json

```
@@ -12,6 +12,7 @@
     "@sitespeed.io/geckodriver": "0.32.2",
     "@sitespeed.io/throttle": "5.0.0",
     "@sitespeed.io/tracium": "0.3.3",
+    "bidi-har-export": "firefox-devtools/bidi-har-export#b92585f39b6640ec93e57b0395b4dc22c2bde666",
```

@soulgalore can you try with 55179f52bb77ea103569bb19ededef4c78d84048
```suggestion
    "bidi-har-export": "firefox-devtools/bidi-har-export#55179f52bb77ea103569bb19ededef4c78d84048",
```